pandas/tests/scalar/period/test_period.py

```diff
@@ -1148,6 +1148,16 @@ def test_period_cmp_nat(self):
             assert not left <= right
             assert not left >= right
 
+    @pytest.mark.parametrize(
+        "scalar, expected",
+        ((np.array(0), False), (np.int_(0), False), (Period("2000-01", "M"), True)),
+    )
+    def test_comparison_numpy_scalar(self, scalar, expected):
```

Comment:
scalars are already tested; its zerodim we need to catch.  in particular a zero-dim ndarray containing a Period object